var/spack/repos/builtin/packages/py-biom-format/package.py

```diff
@@ -27,4 +29,4 @@ class PyBiomFormat(PythonPackage):
     depends_on('py-scipy@0.13.0:', type=('build', 'run'))
```

Comment:
Numpy is now `@1.9.2:`

Reply:
Version 2.1.6 requires numpy >= 1.3.0, should I always default to the latest version if possible?